pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -216,6 +216,16 @@ class TestTimedeltaMultiplicationDivision(object):
     # ---------------------------------------------------------------
     # Timedelta.__mul__, __rmul__
 
+    @pytest.mark.parametrize('td_nat', [pd.NaT,
+                                        np.timedelta64('NaT', 'ns'),
+                                        np.timedelta64('NaT')])
+    @pytest.mark.parametrize('op', [operator.mul, ops.rmul])
+    def test_td_mul_nat(self, op, td_nat):
+        # GH#19819
+        td = Timedelta(10, unit='d')
+        with pytest.raises(TypeError):
```

Comment:
why do we raise on this again? what is the behavior on divide?

Reply:
For the same reason we raise on `Timedelta * timedelta` (or `Timedelta * datetime` depending on what hat `pd.NaT` is wearing).

For division `Timedelta / pd.NaT` returns `np.nan`, which is already correct.